### DISCUSSION TITLE
What is the best setting for using Niji-Journey V5 Expressive Style?

### DISCUSSION THREAD

**Discussion by Sven-Lyco** [lllyasviel/Fooocus#972] [Q&A] [answered]
Hi folks,
is it possible to render with images like in Niji-Journey V5 with expressive style and what could be the best setting for this?

The styling of the images should look like this:
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/d082492d-0c5d-459e-82d6-8fee26a326c9" height="500">

**Comment by xhoxye** [comment]
Change the Niji SDXL model.

**Reply by Sven-Lyco** [reply]
Can you tell me how do I do this? I don't have that much experience.

**Reply by xhoxye** [reply]
![QQ截图20231118185646](https://github.com/lllyasviel/Fooocus/assets/129571231/5c23ce5b-f8f9-455f-aadc-772b9f873568)
![QQ截图20231118185706](https://github.com/lllyasviel/Fooocus/assets/129571231/54b2d46d-9e04-41aa-8858-ab080bfa3f71)

**Reply by Sven-Lyco** [reply]
These are the results:
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/13e05ae1-a66c-4ba9-b4bd-c5da4a3b5aad" height="500">
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/c2049f63-0c98-4864-918a-b63cce43e121" height="500">
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/8a2beb10-c8b1-4433-8a2b-0347f89b53b5" height="500">
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/3c507c96-091c-4ae2-b05a-09cdb0f96b79" height="500">

and is use NijiSpecial. Thats not really what i wanted 😅
Any suggestion, please?🙏🏻

**Comment by xhoxye** [comment, marked as answer]
![QQ截图20231120175308](https://github.com/lllyasviel/Fooocus/assets/129571231/50626fc7-ecd6-404d-9387-ad47c8648d81)

**Reply by Sven-Lyco** [reply]
That looks awesome, can you share the whole log of this picture please?

**Comment by xhoxye** [comment]
![QQ截图20231120175941](https://github.com/lllyasviel/Fooocus/assets/129571231/00c6d850-428d-46cb-84ea-351a7f941944)
Prompt: A girl sits at the bar counter of a café with several people in the background and a light,Warm and elegant atmosphere
Fooocus V2 Expansion: A girl sits at the bar counter of a café with several people in the background and a light,Warm and elegant atmosphere, crisp intricate, highly detailed, candid, beautiful, refined, artistic, complex, sharp focus, inspired, innocent, pretty, dramatic, illuminated color, cinematic, professional, protected, attractive, delicate, extremely fine detail, full, coherent, magic

Styles: ['Fooocus V2', 'SAI Anime', '♡动漫预设 2d平面♡', '♡3D 迪士尼 Disney'], Performance: Speed

Resolution: (1152, 896), Sharpness: 2

Guidance Scale: 4, ADM Guidance: (1.5, 0.8, 0.3)

Base Model: SDXL\sdxlNijiSpecial_sdxlNijiSE.safetensors, Refiner Model: None

Refiner Switch: 1, Sampler: dpmpp_2m_sde_gpu

Scheduler: karras, Seed: 1179072762071872870

LoRA [sd_xl_offset_example-lora_1.0.safetensors] weight: 0.1,

**Reply by Sven-Lyco** [reply]
Thank you, can you share the log from the image above too please?  That would be great

**Reply by xhoxye** [reply]
It's all in the text above, a sentence I wrote casually

**Reply by Sven-Lyco** [reply]
Yes i saw it, but  i mean this one. https://github.com/lllyasviel/Fooocus/discussions/972#discussioncomment-7617614 
I see theres a refiner defined, and the text is cutted.

**Reply by xhoxye** [reply]
DreamShaper_8_pruned.safetensors，It doesn't matter, you can have it or not

**Reply by Sven-Lyco** [reply]
It would be nice, if you can share it. I would give it a try if can make the same result.

**Comment by Sven-Lyco** [comment]
So i got i now, thanks @xhoxye for the help.
I have noticed that I have used a different base model. I had version 5 and not SE, from https://civitai.com/models/120765/sdxlnijispecial-edition.
And I tried something around with the settings, based on your two pictures and here is one of the result:
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/5de435b9-5d1e-4e31-bd5f-aab6823cfe98" height="400">
<img src="https://github.com/lllyasviel/Fooocus/assets/98020906/287946a7-d787-4690-a953-6c8aa18f898b" height="500">
